client/landing/gutenboarding/onboarding-block/designs/index.tsx

```diff
@@ -1,24 +1,80 @@
-import DesignPicker, { getAvailableDesigns } from '@automattic/design-picker';
-import { useLocale } from '@automattic/i18n-utils';
+import { isEnabled } from '@automattic/calypso-config';
+import DesignPicker, {
+	FeaturedPicksButtons,
+	getAvailableDesigns,
+	useCategorization,
+} from '@automattic/design-picker';
+import { useLocale, englishLocales } from '@automattic/i18n-utils';
 import { Title, SubTitle, ActionButtons, BackButton } from '@automattic/onboarding';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { useI18n } from '@wordpress/react-i18n';
-import { useEffect } from 'react';
+import classnames from 'classnames';
+import { useEffect, useMemo } from 'react';
 import * as React from 'react';
 import JetpackLogo from 'calypso/components/jetpack-logo'; // @TODO: extract to @automattic package
 import Badge from '../../components/badge';
 import useStepNavigation from '../../hooks/use-step-navigation';
 import { useTrackStep } from '../../hooks/use-track-step';
 import { useIsAnchorFm } from '../../path';
 import { STORE_KEY as ONBOARD_STORE } from '../../stores/onboard';
-import type { Design } from '@automattic/design-picker';
+import type { Design, Category } from '@automattic/design-picker';
 
 import './style.scss';
 
+// Ensures Blog category appears at the top of the design category list
+// (directly below the All Themes category).
+const sortBlogToTop = ( a: Category, b: Category ) => {
+	if ( a.slug === b.slug ) {
+		return 0;
+	} else if ( a.slug === 'blog' ) {
+		return -1;
+	} else if ( b.slug === 'blog' ) {
+		return 1;
+	}
+	return 0;
+};
+
+const Header: React.FunctionComponent = () => {
+	const { __ } = useI18n();
+	const locale = useLocale();
+	const isAnchorFmSignup = useIsAnchorFm();
+	const isDesignPickerCategoriesEnabled =
+		isAnchorFmSignup && isEnabled( 'signup/design-picker-categories' );
```

Comment:
The categories didn't appear until I made these changes locally.

```suggestion
		! isAnchorFmSignup && isEnabled( 'signup/design-picker-categories' );
```

Reply:
Oops...my brain got stuck at that time 🥲